sftp-client.c

```diff
@@ -1193,6 +1222,41 @@ do_download(struct sftp_conn *conn, const char *remote_path,
 
 	attrib_clear(&junk); /* Send empty attributes */
 
+#ifdef IRONCORE
+	if (strlen(local_path) > PATH_MAX - IRON_SECURE_FILE_SUFFIX_LEN) {
+		error("Local path name %s too long.", local_path);
```

Comment:
Typically you have been putting path names inside of `"..."` within error messages.